projects/aca-content/src/lib/components/toolbar/toggle-edit-offline/toggle-edit-offline.component.ts

```diff
@@ -94,6 +96,7 @@ export class ToggleEditOfflineComponent implements OnInit {
         this.update(response?.entry);
         this.store.dispatch(new DownloadNodesAction([this.selection]));
         this.store.dispatch(new EditOfflineAction(this.selection));
+        this.store.dispatch(new SetSelectedNodesAction([this.selection]));
```

Comment:
Maybe we can move this statement outside of the if-else block, since it is being used in both if and else conditions.

Reply:
@swapnil-verma-gl I put those inside if-else, because there is no need to update toolbar menu actions refs if an error occurs during locking/unlocking. I've just tested that dispatching the `SetSelectedNodesAction` after lock error does not update file state, reloading file list is needed anyway.